libres/tests/enkf/enkf_obs_paths_detailed.cpp

```diff
@@ -0,0 +1,179 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <string>
+#include <iostream>
+
+#include <catch2/catch.hpp>
+
+#include <ert/util/test_work_area.hpp>
+#include <ert/util/test_util.h>
+
+#include <ert/enkf/enkf_obs.hpp>
+#include "../tmpdir.hpp"
+
+/*
+ * Set up workarea, create and write start of file.
+ * Returns pointer to workares since it needs to be kept alive
+ * for the actual test
+ */
+void setup() {
+    util_make_path("obs_path");
+    FILE *stream = fopen("obs_path/conf.txt", "w");
```

Comment:
Should we go all in on C++ and use  `ofstream` etc., or is it not worth the trouble?